config/routes.rb

```diff
@@ -39,6 +39,11 @@
   get "/login", to: "sessions#new"
 
   get "/profile", to: "profile#index"
+  get "/profile/:id/password", to: "passwords#edit"
+  get "/profile/:id/edit", to: "profile#edit"
+  patch "/profile/:id/edit", to: "profile#update"
+  patch "/profile/:id/password", to: "passwords#update"
+
```

Comment:
Note: We didn't namespace the 'passwords' routes. May want to consider doing so for consistency with other profile/user access.